How to fill a DeleteRulesetRuleParams

### Confirmation

- [X] My issue isn't already found on the issue tracker.
- [X] I have replicated my issue using the latest version of the library and it is still present.

### cloudflare-go version

v0.101.1-0.20240809030210-5dc300a49857

### Go environment

```
GO111MODULE=''
GOARCH='arm64'
GOBIN='/Users/tlimoncelli/bin'
GOCACHE='/Users/tlimoncelli/Library/Caches/go-build'
GOENV='/Users/tlimoncelli/Library/Application Support/go/env'
GOEXE=''
GOEXPERIMENT=''
GOFLAGS=''
GOHOSTARCH='arm64'
GOHOSTOS='darwin'
GOINSECURE=''
GOMODCACHE='/Users/tlimoncelli/go/pkg/mod'
GONOPROXY=''
GONOSUMDB=''
GOOS='darwin'
GOPATH='/Users/tlimoncelli/go'
GOPRIVATE=''
GOPROXY='https://proxy.golang.org,direct'
GOROOT='/usr/local/go'
GOSUMDB='sum.golang.org'
GOTMPDIR=''
GOTOOLCHAIN='auto'
GOTOOLDIR='/usr/local/go/pkg/tool/darwin_arm64'
GOVCS=''
GOVERSION='go1.22.5'
GCCGO='gccgo'
AR='ar'
CC='clang'
CXX='clang++'
CGO_ENABLED='1'
GOMOD='/Users/tlimoncelli/tmp/cloudflare-go/go.mod'
GOWORK=''
CGO_CFLAGS='-O2 -g'
CGO_CPPFLAGS=''
CGO_CXXFLAGS='-O2 -g'
CGO_FFLAGS='-O2 -g'
CGO_LDFLAGS='-O2 -g'
PKG_CONFIG='pkg-config'
GOGCCFLAGS='-fPIC -arch arm64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -ffile-prefix-map=/var/folders/lx/4w8fz8z916g911yvplhy6jjc0000gp/T/go-build1864510889=/tmp/go-build -gno-record-gcc-switches -fno-common'
```

### Expected output

In https://github.com/cloudflare/cloudflare-go/pull/2833. the type DeleteRulesetRuleParams struct was added:

```
type DeleteRulesetRuleParams struct {
	rulesetID     string `json:"-"`
	rulesetRuleID string `json:"-"`
}
```

How does one instantiate this struct?  The fields are unexported.

We tried this:

```
  p := cloudflare.DeleteRulesetRuleParams{cfr.SRRRulesetID, cfr.SRRRulesetRuleID}
  err := c.cfClient.DeleteRulesetRule(context.Background(), cloudflare.ZoneIdentifier(domainID), p)
```

but Go complains:

```
$ go build
# github.com/StackExchange/dnscontrol/v4/providers/cloudflare
providers/cloudflare/rest.go:391:42: implicit assignment to unexported field rulesetID in struct literal of type "github.com/cloudflare/cloudflare-go".DeleteRulesetRuleParams
providers/cloudflare/rest.go:391:60: implicit assignment to unexported field rulesetRuleID in struct literal of type "github.com/cloudflare/cloudflare-go".DeleteRulesetRuleParams
```

I feel like I'm missing something.  Should the fields me exported or is there a constructor I'm unaware of?  The other `*Params` structs export all their fields.

Thanks!

CC [jacobbednarz](https://github.com/jacobbednarz) for visibility.

### Actual output

see above

### Code demonstrating the issue


```
  p := cloudflare.DeleteRulesetRuleParams{cfr.SRRRulesetID, cfr.SRRRulesetRuleID}
  err := c.cfClient.DeleteRulesetRule(context.Background(), cloudflare.ZoneIdentifier(domainID), p)
```


### Steps to reproduce

see above

### References

_No response_

--- a/.changelog/2886.txt
+++ b/.changelog/2886.txt
@@ -1,0 +1,3 @@
+```release-note:enhancement
+rulesets: Export `DeleteRulesetRuleParams` fields
+```

--- a/rulesets.go
+++ b/rulesets.go
@@ -854,7 +854,7 @@ func (api *API) UpdateRuleset(ctx context.Context, rc *ResourceContainer, params
 //
 // API reference: https://developers.cloudflare.com/api/operations/deleteZoneRulesetRule
 func (api *API) DeleteRulesetRule(ctx context.Context, rc *ResourceContainer, params DeleteRulesetRuleParams) error {
-	uri := fmt.Sprintf("/%s/%s/rulesets/%s/rules/%s", rc.Level, rc.Identifier, params.rulesetID, params.rulesetRuleID)
+	uri := fmt.Sprintf("/%s/%s/rulesets/%s/rules/%s", rc.Level, rc.Identifier, params.RulesetID, params.RulesetRuleID)
 	res, err := api.makeRequestContext(ctx, http.MethodDelete, uri, nil)
 	if err != nil {
 		return err

--- a/rulesets.go
+++ b/rulesets.go
@@ -742,8 +742,8 @@ type UpdateEntrypointRulesetParams struct {
 }
 
 type DeleteRulesetRuleParams struct {
-	rulesetID     string `json:"-"`
-	rulesetRuleID string `json:"-"`
+	RulesetID     string `json:"-"`
+	RulesetRuleID string `json:"-"`
 }
 
 // ListRulesets lists all Rulesets in a given zone or account depending on the

